src/Features/Core/Portable/ChangeSignature/AbstractChangeSignatureService.cs

```diff
@@ -81,39 +79,14 @@ internal abstract class AbstractChangeSignatureService : ILanguageService
 
         public async Task<ImmutableArray<ChangeSignatureCodeAction>> GetChangeSignatureCodeActionAsync(Document document, TextSpan span, CancellationToken cancellationToken)
```

Comment:
I admit I'm surprised this function is here versus just being inlined into the actual code fix provider. Not urgent to fix though.